### DISCUSSION TITLE
When using parallel desktop to run pksim on MacOS 15 come cross this problem following screenshot

### DISCUSSION THREAD

**Discussion by asayang4real** [Open-Systems-Pharmacology/Forum#1841] [Questions & Problems]
How to deal with it ?
<img width="931" alt="Screenshot 2024-09-24 at 12 15 14" src="https://github.com/user-attachments/assets/e5438f81-2889-40c0-bafd-4adbc068e840">

**Comment by StephanSchaller** [comment]
Where are you running the Software? On a Server? Locally?

**Reply by asayang4real** [reply]
Locally. MacBook Pro, M3 Max, Parallels Desktop Windows 11

**Comment by PavelBal** [comment]
@rwmcintosh did you every try this?

**Comment by rwmcintosh** [comment]
Install latest [VC Redist](https://aka.ms/vs/17/release/vc_redist.x64.exe)
